src/components/common/Button/Button.tsx

```diff
@@ -1,13 +1,14 @@
 import type { FC, PropsWithChildren } from 'react';
 import css from './btn.module.css';
+import cn from 'classnames';
 
 interface ButtonProps {
     onClick: () => void;
     disabled?: boolean;
-    className?: 'addTask' | 'deleteTask';
+    className?: string;
 }
 
 export const Button: FC<PropsWithChildren<ButtonProps>> = ({
-    children, onClick, disabled, className = 'addTask'}) => 
-    <button onClick={ onClick } disabled={ disabled } className = { css[className] } >{ children }</button>
+    children, onClick, disabled, className}) => 
+    <button onClick={ onClick } disabled={ disabled } className = { cn(css.addTask, className) } >{ children }</button>
```

Comment:
```suggestion
    <button onClick={ onClick } disabled={ disabled } className = { cn(css.btn, className) } >{ children }</button>
```
Это комоновская кнопка, которая может использовать где угодно в приложении. Поэтому даем имена максимально абстрактные не привязанные ни к какому контексту использования